articles/azure-monitor/logs/resource-manager-cluster.md

```diff
@@ -267,6 +274,19 @@ resource cluster 'Microsoft.OperationalInsights/clusters@2021-06-01' = {
 
 ### Parameter file
 
+# [Bicep Parameters file](#tab/bicep)
+
+param clusterName = 'MyCluster'
+param location = resourceGroup().location
+param keyVaultName = 'MyKeyVault'
+param keyName = 'MyKeyName'
+param keyVersion = 'desired version'
```

Comment:
This is optional - I believe that in most cases, value would be '', namely empty, and perhaps should be in the example?